packages/better-auth/src/plugins/email-otp/index.ts

```diff
@@ -49,9 +59,9 @@ const types = ["email-verification", "sign-in", "forget-password"] as const;
 export const emailOTP = (options: EmailOTPOptions) => {
 	const opts = {
 		expiresIn: 5 * 60,
-		otpLength: 6,
+		generateOTP: () => generateRandomString(options.otpLength ?? 6, "0-9"),
```

Comment:
this here would be the alternative to remove length (change is in the last commit)